Maven build with noredist and simulator flags failing

Build on main branch is failing with:

```
$ mvn clean install -P developer,systemvm -Dnoredist -Dsimulator  -T8
...
[ERROR] Failed to execute goal on project cloud-plugin-network-nsx: Could not resolve dependencies for project org.apache.cloudstack:cloud-plugin-network-nsx:jar:4.20.0.0-SNAPSHOT: The following artifacts could not be resolved: com.vmware.vapi:vapi-samltoken:jar:2.40.0 (absent): com.vmware.vapi:vapi-samltoken:jar:2.40.0 was not found in https://repo.jenkins-ci.org/releases/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of repo.jenkins-ci.org.releases has elapsed or updates are forced -> [Help 1]
[ERROR]
```

##### ISSUE TYPE
<!-- Pick one below and delete the rest -->
 * Bug Report

##### COMPONENT NAME
<!--
Categorize the issue, e.g. API, VR, VPN, UI, etc.
-->
~~~
Build
~~~

##### CLOUDSTACK VERSION
<!--
New line separated list of affected versions, commit ID for issues on main branch.
-->

~~~
main branch (4.20.0.0-snapshot)
~~~

cc @weizhouapache @nvazquez @Pearl1594

My env details are:
```
Apache Maven 3.9.6 (bc0240f3c744dd6b6ec2920b3cd08dcc295161ae)
Maven home: /opt/homebrew/Cellar/maven/3.9.6/libexec
Java version: 11.0.16.1, vendor: Homebrew, runtime: /opt/homebrew/Cellar/openjdk@11/11.0.16.1_1/libexec/openjdk.jdk/Contents/Home
Default locale: en_IN, platform encoding: UTF-8
OS name: "mac os x", version: "14.4.1", arch: "aarch64", family: "mac"
```

Thanks all, I could run this again today and same error for me:
```
mvn clean install -P developer,systemvm -Dnoredist -Dsimulator  -T8
...
[ERROR] Failed to execute goal on project cloud-plugin-network-nsx: Could not resolve dependencies for project org.apache.cloudstack:cloud-plugin-network-nsx:jar:4.20.0.0-SNAPSHOT: The following artifacts could not be resolved: com.vmware.vapi:vapi-samltoken:jar:2.40.0 (absent): Could not find artifact com.vmware.vapi:vapi-samltoken:jar:2.40.0 in repo.jenkins-ci.org.releases (https://repo.jenkins-ci.org/releases/) -> [Help 1]
```

On debugging further, it's coming as indirect dependency of:
```
 985 [DEBUG]    com.vmware.vapi:vapi-authentication:jar:2.40.0:compile
 986 [DEBUG]       com.vmware.vapi:vapi-samltoken:jar:2.40.0:compile                                                                                                          987 [DEBUG]       org.slf4j:slf4j-api:jar:1.7.29:compile
```

--- a/plugins/network-elements/nsx/pom.xml
+++ b/plugins/network-elements/nsx/pom.xml
@@ -49,6 +49,12 @@
             <groupId>com.vmware.vapi</groupId>
             <artifactId>vapi-authentication</artifactId>
             <version>2.40.0</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>com.vmware.vapi</groupId>

